test/provider/model/ClientRegistry.unittest.ts

```diff
@@ -1,35 +1,34 @@
-import 'jest';
-import 'fake-indexeddb/auto';
+import 'reflect-metadata';
 
 import {Identity} from 'openfin/_v2/main';
+import {Signal} from 'openfin-service-signal';
 
 import {ClientRegistry} from '../../../src/provider/model/ClientRegistry';
 import {StoredApplication} from '../../../src/provider/model/Environment';
-import {createMockEnvironment, createMockApiHandler, createMockServiceStore} from '../../mocks';
-import {PartiallyWritable} from '../../types';
+import {createMockEnvironment, createMockApiHandler, createMockServiceStore, getterMock} from '../../utils/unit/mocks';
 import {RegisterApplication} from '../../../src/provider/store/Actions';
+import {createFakeStoredApplication, createFakeUuid, createFakeEmptyWebWindow, createFakeIdentity} from '../../utils/unit/fakes';
+import {resolvePromiseChain} from '../../utils/unit/time';
 
-jest.unmock('../../../src/provider/model/ClientRegistry');
+const mockApiHandler = createMockApiHandler();
+const mockEnvironment = createMockEnvironment();
+const mockServiceStore = createMockServiceStore();
 
 beforeEach(async () => {
     jest.resetAllMocks();
+
+    getterMock(mockApiHandler, 'onConnection').mockReturnValue(new Signal<[Identity]>());
+    getterMock(mockApiHandler, 'onDisconnection').mockReturnValue(new Signal<[Identity]>());
 });
 
 describe('When attemping to launch an app through the client registry', () => {
```

Comment:
while you're here :) attemping -> attempting

Reply:
✅